modules/post/multi/recon/local_exploit_suggester.rb

```diff
@@ -60,8 +72,36 @@ def is_module_platform?(mod)
     return false
   end
 
-  def is_session_type_compat?(mod)
-    mod.session_compatible? session.sid
+  def has_required_module_options?(mod)
+    mod.options.each_pair do |option_name, option|
+      return false if option.required && option.default.nil? && mod.datastore[option_name].blank?
+    end
+
+    true
+  end
+
+  def get_all_missing_module_options(mod)
```

Comment:
Not a blocker, normally Ruby methods aren't prefixed with `get_` as it's more of a Java-ism